client/src/components/ChartBars.vue

```diff
@@ -12,29 +12,47 @@ export default {
     rounds: {
       type: Array,
       required: true
+    },
+    tab: {
+      type: String,
+      required: true
     }
   },
+  computed: {
+    chartdata () {
+      return {
+          labels: this.team.coop ? ["Victories", "Defeats"] : this.getPlayers(),
+          datasets: [
+            {
+              label: "Victories",
+              backgroundColor: "#ec8506",
+              data: this.team.coop ? this.getCoopStat() : this.getPlayersStat()
+            }
+          ]
+        }
+    },
+  },
   watch: {
-    rounds(value) {
-      value ? this.setChart() : this._chart.destroy();
+    rounds() {
+      this.updateChart()
+    },
+    tab(val) {
+      if (val === 'tab-1') this.updateChart()
```

Comment:
Can we omit this strong coupling with tabs? This component could be used as a standalone in the future.

Reply:
Jup, you're right, I will remove it. It was redundant anyways, fix works without it.